src/main/kotlin/com/kotlin/boardproject/domain/notification/NotificationController.kt

```diff
@@ -1,17 +1,17 @@
-package com.kotlin.boardproject.controller
+package com.kotlin.boardproject.domain.notification
 
-import com.kotlin.boardproject.auth.LoginUser
-import com.kotlin.boardproject.dto.common.ApiResponse
-import com.kotlin.boardproject.dto.notification.GetNotificationsResponseDto
-import com.kotlin.boardproject.service.NotificationService
+import com.kotlin.boardproject.domain.notification.dto.GetNotificationsResponseDto
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end